lib/jsdom/living/nodes/HTMLTemplateElement-impl.js

```diff
@@ -43,7 +43,7 @@ class HTMLTemplateElementImpl extends HTMLElementImpl {
   // https://html.spec.whatwg.org/multipage/scripting.html#template-adopting-steps
   _adoptingSteps() {
     const doc = this._appropriateTemplateContentsOwnerDocument(this._ownerDocument);
-    doc._adoptNode(this._templateContents);
+    doc._adoptNode(this._templateContents, /* doItAnyway= */ true);
```

Comment:
```suggestion
    doc._adoptNode(this._templateContents, /* forceDocumentFragmentAdoption= */ true);
```

Reply:
There should be a space between `forceDocumentFragmentAdoption` and `=`.

```suggestion
    doc._adoptNode(this._templateContents, /* forceDocumentFragmentAdoption = */ true);
```

Reply:
@ExE-Boss, we've already discussed how unsolicited reviews are not welcome. They are especially unwelcome when they are on the PRs of contributors, instead of core members; we do not want the jsdom project to be a place where submitting a PR means you get random people coming by and critiquing your work, disconnected from the project maintainers.